### PR TITLE
Vtorri lot export

### PR DESCRIPTION
**Pull request by vtorri**

### CODE REVIEW

**Comment by smohantty** [comment]
@vtorri , please rebase it to latest .
undef LOT_EXPORT if defined . is it necessary ?

File: inc/rlottie.h [hunk 1]
```diff
@@ -451,4 +455,7 @@ template<> struct MapType<std::integral_constant<Property, Property::TrScale>>:
 
 }  // namespace lotplayer
 
+#undef LOT_EXPORT
```
[the remark this answers is not in the document]
Comment (vtorri) [reply inc/rlottie.h]: same as above, i've follow the EAPI scheme and it's for including header files defining LOT_EXPORT in any order. Maybe this one can be removed

**Comment by vtorri** [comment]
i'll redo this pull request, i can't remove these merges

**Comment by smohantty** [comment]
ok

**Pull request closed without merging**